ServicemanUnitTest/TestOfUnit.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ArmyLibrary;
+
+namespace ServicemanUnitTest;
+
+public class UnitTests
```

Comment:
Класс с тестами попал вне пространства имен и не виден обозревателю тестов
```suggestion
namespace ArmyLibrary.UnitTests
```